controllers/userController.js

```diff
@@ -1,6 +1,7 @@
 const admin = require('firebase-admin');
+const { firebaseDb } = require("../config/firebaseConfig");
 const signUp = async (req, res) => {
-  const { email, password } = req.body;
+  const { email, password, address, age, gender, name } = req.body;
```

Comment:
Fields should be firstName, lastName, MobileNo, email, password, confirmPassword, profilePhoto.